shell/common/test_vsync_waiters.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#define FML_USED_ON_EMBEDDER
+
+#ifndef FLUTTER_SHELL_COMMON_TEST_VSYNC_WAITERS_H_
+#define FLUTTER_SHELL_COMMON_TEST_VSYNC_WAITERS_H_
+
+#include "flutter/shell/common/shell.h"
+
+namespace flutter {
+namespace testing {
+
+class ShellTestVsyncClock {
```

Comment:
IMO, it is better to have one TU per class as it makes code easier to navigate (just lookup the the file name for the class you want to navigate to). But your call.

Reply:
I have a few more changes coming up. Will move things around to their own files.